examples/util/BaseRunner.js

```diff
@@ -165,145 +79,15 @@ class BaseRunner {
   };
 
   loadModel = async (modelInfo) => {
```

Comment:
For `loadMode`, it would be good to have some comments here that document the input, output and behavior of this function. So developer has the idea about how to override it.

Reply:
Thanks @huningxin.
Yes, a guide for how to use these core methods is necessary and helpful for later example contributors. I will document it and submit the new PR, thanks.

Reply:
Let's do it in light way. How about add comment to these methods in BaseRunner? It would be also helpful to review this PR.

Reply:
OK, got it, thanks.